bundles/org.openhab.core.io.rest.auth/src/main/resources/OH-INF/i18n/restauth_fr.properties

```diff
@@ -0,0 +1,8 @@
+system.config.restauth.allowBasicAuth.label = Autoriser l'authentification basique
+system.config.restauth.allowBasicAuth.description = Autoriser l'utilisation de l'authentification basique pour accéder aux ressources d'API protégées, ainsi que l'accès aux jetons et jetons d'API.
+system.config.restauth.cacheExpiration.label = Temps d'expiration du cache
+system.config.restauth.cacheExpiration.description = Lorsque l'authentification de base est activée, les identifiants sont placés en cache afin d'accélérer les requêtes d'autorisations. Les entrées dans le cache expirent après un certain temps afin de ne pas conserver les informations d'identification en mémoire indéfiniment. Cette valeur définit le temps d'expiration en heures. Définissez-le à 0 pour désactiver le cache.
+system.config.restauth.implicitUserRole.label = Rôle implicite d'utilisateur
+system.config.restauth.implicitUserRole.description = Par défaut, les opérations nécessitant le rôle "user" sont disponibles lorsqu'elles ne sont pas authentifiées. Désactiver cette option imposera l'autorisation pour ces opérations. Avertissement\: Les clients qui ne prennent pas en charge l'authentification peuvent cesser de fonctionner.
+
+service.system.restauth.label = API de sécurité
```

Comment:
This is a really bad translation for "API Security". I noticed it yesterday and changed it on Crowdin but it didn't make it to this PR, can you make sure it doesn't get into the milestone build? ;)

Reply:
@ghys This one: https://github.com/openhab/openhab-core/pull/2390?

Reply:
Yup, much better, thank you! I thought sync of approved strings happened automatically.

Reply:
Yes, once a day. But you can trigger a manual sync too. I forgot to do that before merging this PR. Will try to remember next time.